### DISCUSSION TITLE
Still seeing `Recent push is still being processed, and will show up in a bit` several hours after a push to PR branch

### DISCUSSION THREAD

**Discussion by rm3l** [community/community#78775] [Pull Requests]
### Select Topic Area

Bug

### Body

Hi there,

I'm working on this PR: https://github.com/janus-idp/operator/pull/56

I pushed some changes to the corresponding branch, but they still do not show up in the PR view, which blocks people from reviewing the PR. It has been so for several hours now.

![image](https://github.com/community/community/assets/593208/c17117bf-a4f7-4870-9fe9-79a728183f79)

Any clue about what could be wrong here?

I noticed the same issue on another PR in the same repo.

Thanks.

**Comment by rm3l** [comment]
We ended up recreating the PRs.

**Comment by dubrova-ta-ma** [comment]
same for me, recreating PRs helps

**Comment by ivan-klass** [comment]
First time faced this issue. Commit is there in branch and actions are executed for push event, but not yet shown in PR after 4 hours

**Comment by Bhushan-WhiteBoxData** [comment]
Anyone found a solution to this? Been waiting for a long time and do not know what to do?
Issue is Merge isn't working for me. It is only the last commit that is not showing up. And at this point do not know how to get to a point where it completes and we are able to merge

**Reply by fuzzdeveloper** [reply]
If it makes you feel better it might be an issue with GH right now because I'm experiencing it too (noticed you posted this 22 min ago which is about when i pushed my changes).

**Reply by Bhushan-WhiteBoxData** [reply]
Thanks for letting me know. Definitely feel slightly better and really hoping its just that. I will let it be and hope it gets resolved on its own. We had multiple commits today and the last one was the one that caused an issue so wasn't sure if it was just me. 🤞

**Comment by chesnokoff** [comment]
Same. I've been waiting for 3 hours and it's still processing. It seems like recreating the PR is the only solution.

**Comment by pbratkowski** [comment]
I got around this by `git commit --amend` and `git push --force` with no new changes, which is a very hacky and silly way to get around an apparent bug.

**Reply by schmidti159** [reply]
Just had the same issue and pushing an empty commit fixed it for me as well.

**Reply by varonix0** [reply]
Just had this happen as well and the above fixed it for me. Based on reports this seems to be more frequent now than it used to be

**Reply by andrescerv** [reply]
`git commit --amend` and `git push --force` with no new changes did it for me

**Comment by dsessler7** [comment]
The top answer on this stackoverflow did the trick for me:
https://stackoverflow.com/questions/45626986/updating-pull-request-after-review-changes-commit-doesnt-show-up-in-the-pull-re

**Reply by Dirreke** [reply]
> The top answer on this stackoverflow did the trick for me: https://stackoverflow.com/questions/45626986/updating-pull-request-after-review-changes-commit-doesnt-show-up-in-the-pull-re

I met the same Issue today, and the answer works for me.

**Comment by chetanbb-astuto** [comment]
ITS 2026 AND STILL FACING THE SAME ISSUE :)

**Reply by AleksejsKolotovs** [reply]
still for me today! :(

**Reply by chetanbb-astuto** [reply]
@AleksejsKolotovs  Create a new PR it will resolve or follow the stack overflow method mentioned above

**Comment by MiguelERuiz** [comment]
Same issue here :face_exhaling:

**Reply by AleksejsKolotovs** [reply]
https://www.githubstatus.com/ GitHub has some issues now :( - "We are investigating reports of degraded performance for API Requests, Git Operations, Issues and Pull Requests"

**Comment by Surajvatsya** [comment]
`git commit --amend` and `git push --force` with no new changes did it for me -> worked for me :)
